frontend/app/routes/$lang+/_public+/apply+/$id+/confirmation.tsx

```diff
@@ -225,21 +235,23 @@ export default function ApplyFlowConfirm() {
         <li>{t('confirm.dental-public', { access: dentalInsurance.selectedBenefits })}</li>
       </UnorderedList>
 
-      <button
-        className="mt-8 inline-flex w-44 items-center justify-center rounded bg-gray-800 px-5 py-2.5 align-middle font-lato text-xl font-semibold text-white outline-offset-2 hover:bg-gray-900 print:hidden"
+      <Button
+        className="mt-5 print:hidden"
+        size="lg"
+        variant="primary"
         onClick={(event) => {
           event.preventDefault();
           window.print();
         }}
       >
         {t('confirm.print-btn')}
-      </button>
+      </Button>
 
-      <div className="mt-10 flex flex-wrap items-center gap-3">
-        <ButtonLink variant="primary" onClick={() => sessionStorage.removeItem('flow.state')} to="/apply">
+      <fetcher.Form method="post" noValidate className="mt-5 flex flex-wrap items-center gap-3">
+        <ButtonLink type="submit" variant="primary" onClick={() => sessionStorage.removeItem('flow.state')} size="lg" className="print:hidden" to="/apply">
```

Comment:
Use `Button` instead of `ButtonLink`.

```suggestion
        <Button variant="primary" onClick={() => sessionStorage.removeItem('flow.state')} size="lg" className="print:hidden">
```